clients/java/src/main/java/io/camunda/zeebe/client/impl/command/MigrateProcessInstanceCommandImpl.java

```diff
@@ -115,6 +117,23 @@ public ZeebeFuture<MigrateProcessInstanceResponse> send() {
     return future;
   }
 
+  @Override
+  public CamundaFuture<MigrateProcessInstanceResponse> sendCommand() {
+    final MigrateProcessInstanceRequest request = requestBuilder.build();
+
+    final RetriableClientFutureImpl<
+            MigrateProcessInstanceResponse, GatewayOuterClass.MigrateProcessInstanceResponse>
+        future =
+            new RetriableClientFutureImpl<>(
+                MigrateProcessInstanceResponseImpl::new,
+                retryPredicate,
+                streamObserver -> send(request, streamObserver));
+
+    send(request, future);
+
+    return future;
+  }
+
```

Comment:
🙃 If we keep the new method (see comment further above), it's an exact copy of the `send` method, if I'm not mistaken. Can the deprecated method at least invoke the new method then? But maybe we don't really want a new method, this comment can be ignored then and the new method can be removed again.